Lidgren.Network/NetQueue.cs

```diff
@@ -84,7 +84,7 @@ public int Capacity
 
 		private int GetGrowCapacity()
 		{
-			return (int)Math.Pow(2, Math.Log2(m_items.Length) + 1);
+			return m_items.Length * 2;

```

Comment:
So the probelm is that now if you create the queue with a capacity of 0 it won't resize properly.